{{ cookiecutter.__repo_name }}/README.md

```diff
@@ -11,6 +13,23 @@ pip install -e .
 deepaas-run --listen-ip 0.0.0.0
 ```
 
+## Contributing
```

Comment:
I would put this part on the end of the README, normally the last or almost, see for example https://github.com/mlflow/mlflow